docs/docs/concepts/authorization-tokens.mdx

```diff
@@ -12,46 +12,21 @@ There are a number of ways in which you can authenticate and authorize with Micr
 
 ### Microsoft Entra application used by the CLI for Microsoft 365
 
-CLI for Microsoft 365 gets access to Microsoft 365 through a custom Microsoft Entra application named _PnP Microsoft 365 Management Shell_. If you don't want to consent this application in your tenant, you can use a different application instead.
+CLI for Microsoft 365 gets access to Microsoft 365 through your own Microsoft Entra application. 
 
 :::info
 
-When you decide to use your own Microsoft Entra application, you need to choose the application to be a **public client**. Despite the setting's description, the application will not be publicly accessible. This setting enables the use of the device flow for your own application. Without activating this setting, it is not possible to complete the authentication process. The option is currently only available in the preview blade for managing for Microsoft Entra applications.
+When you create your own Microsoft Entra application, you need to choose the application to be a **public client**. Despite the setting's description, the application will not be publicly accessible. This setting enables the use of the device flow for your own application. Without activating this setting, it is not possible to complete the authentication process. The option is currently only available in the preview blade for managing for Microsoft Entra applications.
```

Comment:
Shall we clarify this section that this setting is only required for interactive use of the CLI? If you use CLI with non-interactive scripts, and login using client credentials or MSI, you don't need this.